group_vars/bibdata/alma_staging.yml

```diff
@@ -74,9 +74,9 @@ rails_app_vars:
   - name: BIBDATA_DATA_DIR
     value: '{{bibdata_data_dir}}'
   - name: SOLR_URL
-    value: 'http://lib-solr8-staging.princeton.edu:8983/solr/catalog-alma-qa'
+    value: 'http://lib-solr8-staging.princeton.edu:8983/solr/catalog-alma-staging'
   - name: SOLR_REINDEX_URL
-    value: 'http://lib-solr8-staging.princeton.edu:8983/solr/catalog-alma-qa'
+    value: 'http://lib-solr8-staging.princeton.edu:8983/solr/catalog-alma-staging1'
```

Comment:
Since we added the alias, shall we change this to `catalog-alma-staging-reindex`?

Reply:
yes we can do that.